pom.xml

```diff
@@ -18,11 +18,11 @@
 
   <properties>
     <java.version>1.8</java.version>
-    <avro.version>1.8.1</avro.version>
+    <avro.version>1.8.2</avro.version>
     <scala.version>2.11.12</scala.version>
     <scala.version.prefix>2.11</scala.version.prefix>
-    <spark.version>2.3.0</spark.version>
-    <parquet.version>1.8.2</parquet.version>
+    <spark.version>2.3.2</spark.version>
+    <parquet.version>1.8.3</parquet.version>
```

Comment:
Can we still not remove the pom exclusions for parquet in this update?

Reply:
That is correct.  Without the pom exclusions we run into the same `org.apache.avro.SchemaParseException: Can't redefine: list` in unit tests

Reply:
That's too bad. For some reason, the --packages does not work for parquet-hadoop in python.

Reply:
Unfortunately, I don't understand why it is necessary for running on CDH Spark and not standalone or EMR, or why it works.

Reply:
I have gone through all possible combinations of potential answers to the python problem, and the only thing that seems to work is setting:
`--conf spark.driver.extraClassPath=/<path_to_jar>/org.apache.parquet_parquet-hadoop-1.8.3.jar`

for pyspark. This means the user has to download the jar first somewhere. I cannot find a cleaner solution.